packages/selectInput/components/SelectInput.tsx

```diff
@@ -42,6 +42,10 @@ export interface SelectInputProps extends React.HTMLProps<HTMLSelectElement> {
    * hintContent is text or a ReactNode that is displayed directly under the input with additional information about the expected input.
    */
   hintContent?: React.ReactNode;
+  /**
+   * Optional icon to be displayed before the input value.
+   */
+  icon?: React.ReactElement<HTMLElement> | string;
```

Comment:
Can we rename this prop to `iconStart`? That's the prop name we use for text inputs and buttons